tools/README.md

```diff
@@ -0,0 +1,75 @@
+# AnDOtools
+
+AnDOtools is a little project that contain two scripts :
+- a script to help users generate an AnDO structure based on a csv file 
+(You can find in the Creator folder one exemple of a CSV file names "exemple.csv")
+- a script to help users to dispay an AnDO structure
+
+
+
+## Installation
+### Dependencies
+
+AnDOChecker requires:
+
+- Python (>= 3.6)
+- Pip3
+- tree
+
+### User installation 
+
+```bash
+> pip install AnDOcreator
+> pip install AnDOviewer
+```
```

Comment:
I think it would be better if everything gets installed at once, when you do "pip install AnDOChecker"...

Reply:
Yes this read me is not up-to-date , so I'll will modify it and put it in the main readme

Reply:
Done correcting

Reply:
ok, thanks!
I think the global README.md would still need some re-organization so that it appears more unified (it still looks like it has two parts, one for the Checker, one for the tools; and I think it'd be better if it could appear as just one part with three executables)... WDYT?

Reply:
@Slowblitz can you do adjustments in the main README file so that it overall looks like it's one project that includes 3 apps?

also, there's a typo... look for "dispay", you'll find it...

Reply:
ping @Slowblitz ;)

Reply:
Done =)

Reply:
ahah ;)
thanks!